projects/plugins/jetpack/extensions/plugins/publicize/components/panel/index.js

```diff
@@ -10,44 +10,29 @@
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { compose } from '@wordpress/compose';
 import { Fragment } from '@wordpress/element';
-import { withDispatch, withSelect } from '@wordpress/data';
 
 /**
  * Internal dependencies
  */
 import PublicizeConnectionVerify from '../connection-verify';
 import PublicizeForm from '../form';
-import PublicizeSettingsButton from '../settings-button';
 import PublicizeTwitterOptions from '../twitter/options';
 
-const PublicizePanel = ( { connections, refreshConnections, prePublish } ) => (
-	<Fragment>
-		{ connections && connections.some( connection => connection.enabled ) && (
```

Comment:
Yep this makes sense, we want to refresh the connections whether we have some currently enabled or not.

Reply:
thanks!